Datadog.Trace.sln

```diff
@@ -371,6 +371,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.Tools.Runner.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NetCoreAssemblyLoadFailureOlderNuGet", "test\test-applications\regression\NetCoreAssemblyLoadFailureOlderNuGet\NetCoreAssemblyLoadFailureOlderNuGet.csproj", "{D79491F0-CA92-439B-98CE-7AF9F57EBEB0}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "calltarget", "calltarget", "{933F1D4B-1216-4BC1-956E-8C30818BAA0F}"
```

Comment:
Not terribly important, but maybe we could call this folder `instrumentation` so it's just a superset of different instrumentation conditions we want to test? Or some other broad name? We already have some AzureAppServices, RateLimter, TracingWithoutLimits applications that don't really fit into the `integrations` folder. What do you think?

Reply:
We can always move these files around in a later PR, I was just thinking about it now since you're adding a new solution folder

Reply:
done in 67125b6